### PR TITLE
ROC-7755-Show updated statement of truth in line with legislation coming into effect 6th April 2020

### PR DESCRIPTION
**Pull request by sudheer-ajax**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-7755

### Change description
Show updated statement of truth in line with legislation coming into effect 6th April 2020

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/features/claim/views/check-and-send.njk [hunk 1]
```diff
@@ -139,7 +139,7 @@
         {{ statementOfTruth(
             form,
             statementOfTruthType = SignatureType.CLAIM,
-            label = 'I believe that the facts stated in this claim are true.',
+            label = 'I believe that the facts stated in this claim are true.<p></p><p>I understand that proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement in a document verified by a statement of truth without an honest belief in its truth.</p>',
```
Comment (andrewwa-kainos) [inline src/main/features/claim/views/check-and-send.njk:142]: We shouldn't be using empty paragraphs to establish vspace. If the need for separate \<p>aragraphs is new, we should refactor or overload the `statementOfTruth()` method to accept an array and have it print out `<p>each line</p>`

Comment (sudheer-ajax) [reply src/main/features/claim/views/check-and-send.njk:142]: additional <p> tag was type ..... was testing something now I removed ... yes we have a new requirement to add paragraphs for statement of truth but it applicable  from the second item in an array if we implement an array ..... for now as its only one additional paragraph then I am adding as parameter .... in future any additional paragraphs comes we go for array approach

**Comment by sudheer-ajax** [comment]
Created another PR for the same changes https://github.com/hmcts/cmc-citizen-frontend/pull/2265
as this one was failing in pipeline.

**Comment by sudheer-ajax** [comment]
These changes went live with duplicate PR  https://github.com/hmcts/cmc-citizen-frontend/pull/2265

**Pull request closed without merging**